### PR TITLE
Zeroize on drop

### PR DESCRIPTION
**Pull request by semenov-vladyslav**
# Description of change

Please write a summary of your changes and why you made them.

## Links to any relevant issues

Be sure to reference any related issues by adding `fixes issue #`.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] Enhancement (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Fix

## How the change has been tested

Describe the tests that you ran to verify your changes.

Make sure to provide instructions for the maintainer as well as any relevant configurations.

## Change checklist

Add an `x` to the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by Thoralf-M — APPROVED** [review]

**Review by thibault-martinez — CHANGES_REQUESTED** [review]
Please apply format

File: src/keys/ternary/seed.rs [hunk 1]
```diff
@@ -46,6 +46,8 @@ impl Zeroize for Seed {
     }
 }
 
+impl ZeroizeOnDrop for Seed {}
```
Comment (thibault-martinez) [inline src/keys/ternary/seed.rs:49]: Can't be derived?

File: src/keys/x25519.rs [hunk 2]
```diff
@@ -16,6 +16,8 @@ use core::convert::TryInto;
 #[cfg(feature = "ed25519")]
 use crate::signatures::ed25519;
 
+use zeroize::{Zeroize, ZeroizeOnDrop};
```
Comment (thibault-martinez) [inline src/keys/x25519.rs]: Sorry I'm nitpicking but can you put the import between core and crate imports please? A new section with newline, to be consistent with our usual format.

File: src/keys/ternary/seed.rs (hunk 1 above)
Comment (semenov-vladyslav) [reply src/keys/ternary/seed.rs:49]: No, derive(ZeroizeOnDrop) does actually impl Drop for the type which is already implemented in SecretDrop.

File: src/keys/x25519.rs (hunk 2 above)
Comment (semenov-vladyslav) [reply src/keys/x25519.rs]: Always forget to format.

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**